addons/road-generator/ui/road_point_edit.gd

```diff
@@ -129,3 +130,11 @@ func _apply_settings_target(target, all:bool) -> void:
 		undo_redo.add_do_method(target, "emit_transform")
 		undo_redo.add_undo_method(target, "emit_transform")
 	undo_redo.commit_action()
+
+
+func _flip_roadpoint(rp:RoadPoint) -> void:
+	var undo_redo = _editor_plugin.get_undo_redo()
+	undo_redo.create_action("Flip RoadPoint")
+	_editor_plugin.subaction_flip_roadpoint(rp, undo_redo)
```

Comment:
Implemented the actual flipping as a "subaction" so in theory it'll be easier for us to create a whole RoadContainer normalizing function in the future which is tolerant to undo/redo.